scripts/vippy/common.py

```diff
@@ -352,36 +354,41 @@ def __init__(self, yaml, data, rel_path, snake_name, is_enum):
         self.yaml = yaml
 
     def __repr__(self):
-        return ("<DataType object (name={0!r}, rel_path={1!r})>"
-                .format(self.name, self.rel_path))
+        return "<DataType object (name={0!r}, rel_path={1!r})>".format(
+            self.name, self.rel_path
+        )
 
     @property
     def name(self):
-        return self.data['_name']
+        return self.data["_name"]
 
     @property
     def csv_name(self):
-        return self.data['csv-header-name']
+        return self.data["csv-header-name"]
 
     @property
     def csv_type(self):
-        return self.data['csv-type']
+        return self.data["csv-type"]
+
+    @property
+    def skip_element_on(self):
```

Comment:
This grabs the new "tag" or label from the YAML, incorporates it as an attribute of this object.

Reply:
question non-blocking

by "skip element on", do you mean "on" as opposed to "off"? or is "on" intended to imply an event (as in, for example, [onmouseover](https://www.w3schools.com/jsref/event_onmouseover.asp))

if the latter, which event? and if the former, is "skip element" ever set to off? and if so, where?

Reply:
It is the latter. The "event" is the run mode, where the YAML is being used to produce either of the CSV or XML rst files. @danny-pease